roles/eups/tasks/main.yml

```diff
@@ -33,17 +33,21 @@
     update: yes
     force: yes
     version: "{{ eups.version }}"
+  register: task_git_clone
 - name: Configure EUPS install
   command: ./configure --with-eups=/data/ics/{{ ansible_hostname }}/products --prefix=/data/ics/{{ ansible_hostname }}/products/eups/{{ eups.version }} chdir=/data/ics/{{ ansible_hostname }}/devel/eups/
+  when: task_git_clone.changed
 - name: Build EUPS
   make:
     chdir: "/data/ics/{{ ansible_hostname }}/devel/eups/"
     target: install
+  when: task_git_clone.changed
 - name: Symlink EUPS default directory
   file:
     src: "/data/ics/{{ ansible_hostname }}/products/eups/{{ eups.version }}"
     dest: "/data/ics/{{ ansible_hostname }}/products/eups/default"
     state: link
+  when: task_git_clone.changed
 - name: Symlink EUPS setups at home
```

Comment:
I'm not sure, but I was wondering this task should be skipped too.

Reply:
this might be fine to run, but I thought this is (should be) a part of configure/make operation...

Reply:
I see.
So do you think this part can be as it is?

Reply:
Yes, and after this, symlink of setups.sh shall be persistent by result of this part, and it will not brake idempotency.

Reply:
I understand. Thank you for explanation.
Then I don't have anything more to comment on it.